### PR TITLE
Revise and simplify Windows install instructions for Python 3.5. Incl…

### PR DESCRIPTION
**Pull request by FlipperPA**
…ude optional instructions for install virtualenv and virtualenvwrapper.

### CODE REVIEW

**Comment by FlipperPA** [comment]
If people could help test these instructions with more Windows configurations, that would be great.

**Comment by tovmeod** [comment]
While you're at it I suggest recommending (maybe mentioning it is recommended or worthwhile but not strictly necessary) updating pip and setuptools immediately after creating the virtualenv ('pip install -U pip setutools')

File: docs/howto/windows.txt [hunk 1]
```diff
@@ -16,52 +19,64 @@ Install Python
...
+In the command prompt, execute the following command::
+
+    pip install django
+
+This will download and install the latest Django release.
 
 After the installation has completed, you can verify your Django installation
 by executing ``django-admin --version`` in the command prompt.
 
+Testing in a Browser
```
Comment (timgraham) [inline docs/howto/windows.txt]: In my opinion, to limit the amount of documentation we maintain we should keep this document limited to things that are Windows specific. This section doesn't seem to meet that criteria.

File: docs/howto/windows.txt [hunk 2]
```diff
@@ -1,14 +1,17 @@
-================================
-How to install Django on Windows
-================================
+==================================================================================
+How to install Django on Windows with Python 3.5, virtualenv and virtualenvwrapper
```
Comment (timgraham) [inline docs/howto/windows.txt]: I think the existing title is okay and doesn't need to describe all the specifics.

File: docs/howto/windows.txt [hunk 3]
```diff
@@ -16,52 +19,64 @@ Install Python
 Django is a Python web framework, thus requiring Python to be installed on your
 machine.
 
-To install Python on your machine go to https://python.org/download/, and
-download a Windows MSI installer for Python. Once downloaded, run the MSI
-installer and follow the on-screen instructions.
+To install Python on your machine go to https://python.org/downloads/. The web
+site should detect that you're on Windows, and offer you a download button for
+the latest Python version. Download the EXE file and run it.
```
Comment (timgraham) [inline docs/howto/windows.txt]: can we use the same terminology as the Python download page, e.g. "executable installer" instead of "EXE file"?

**Comment by FlipperPA** [comment]
I've made the changes suggested - good ideas. Thanks Tim.

**Comment by FlipperPA** [comment]
Yikes, I didn't realize there was a failing test previously from Jenkins, mea culpa. It has been fixed by adding terms to the spelling exceptions.

**Comment by timgraham** [comment]
merged in 1ce11f6064e61af74578c402b9334b6ff38b97b2, thanks!

**Pull request closed without merging**